web/src/test/java/org/springframework/security/web/csrf/CookieCsrfTokenRepositoryTests.java

```diff
@@ -92,27 +93,27 @@ public void saveTokenSecure() {
 	}
 
 	@Test
-	public void saveTokenSecureFlagTrue() {
+	void saveTokenSecureFlagTrue() {
 		this.request.setSecure(false);
-		this.repository.setSecure(Boolean.TRUE);
```

Comment:
I'd recommend leaving these tests as-is and adding new tests to confirm that the new method works. 

The reason is because we need tests that continue to affirm that the deprecated code continues to work.

Reply:
I agree. I added new tests.